src/main/java/uk/gov/justice/digital/delius/data/api/AppointmentRelocateResponse.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class AppointmentRelocateResponse {
+
+    @NotNull
+    private Long appointmentId;
```

Comment:
The response is a bit redundant as you get back the appointment ID originally passed in. Is there anything useful we could return?

Reply:
I see it as a placeholder for attributes to be added as and when required